styles.css

```diff
@@ -0,0 +1,502 @@
+.navbar {
+  overflow: hidden;
+  background-color: #3c3a39;
+  position: relative;
+  width: 375px;
+  height: 92px;
+  top: 0;
+}
+
+.navbar ul {
+  color: white;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  display: flex;
+  width: 100%;
+  align-items: center;
+}
+
+.navbar li {
+  list-style-type: none;
+  padding: 1rem;
+}
+
+.navbar a {
+  flex: 1;
+}
+
+.navbar-logo {
+  flex: 1;
+  font-size: 1.5rem;
+  user-select: none;
+}
+
+.navbar-toggle {
+  display: flex;
+  text-align: end;
+}
+
+.container-fluid {
+  background-image: url('./images/illustration-header-mobile.png');
+  background-position: right 0 bottom -43px;
+  background-repeat: no-repeat;
+  background-size: 135px;
+  color: #fff;
+  padding: 0;
+  background-color: #1c1a19;
+  align-items: flex-start;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+}
+
+.element {
+  display: flex;
+  color: dark;
+  list-style: none;
+  gap: 1.5rem;
+  position: absolute;
+  padding-top: 1.2rem;
+}
+
+.home {
+  color: white;
+  display: flex;
+  flex: column;
+  justify-content: center;
+}
+
+.heading-home {
+  color: #ff6b00;
+  margin-top: 1.2rem;
+  margin-top: 120px;
```

Comment:
[OPTIONAL]

- [ ] You have duplicated margins here, please leave only one 😉 I suggest that you give it margin 0 as it is causing a white space in your design (this happens after you remove position absolute from the parent container, which is the correct thing to do).
![image](https://user-images.githubusercontent.com/20567503/176990336-1ffb7c57-5a81-45b7-a9f8-16fc02a4cdee.png)